### ISSUE TITLE
1.7.10 Ender Teleporter shows in NEI but has no recipe  

### ISSUE THREAD

**Issue by VinhDucanh** [crazysnailboy/VanillaTeleporter#9]
Does Ender Teleporter block actually exist in 1.7.10 ?
Mod is loaded in Server and in Client
Basic Teleporter is working fine.

**Comment by crazysnailboy** [comment]
Yes it does exist but apparently I forgot to add the crafting recipe for it! Thanks for pointing it out - I'll have a new version up shortly to fix it.
